content/issues/2023-04-12-push-to-distgit-issue.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Issue with pushing to distgit"
```

Comment:
```suggestion
title: "Issue with pushing to dist-git"
```

Reply:
We should write down how we spell things like dist-git, spec-file,...;)